sushi-config.yaml

```diff
@@ -4,13 +4,16 @@
 # ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
 id: hl7.fhir.uv.smart
 canonical: http://fhir-registry.smarthealthit.org
-name: smart-app-launch
+name: SmartAppLaunch
```

Comment:
Can you help me understand the reason behind this change?

Reply:
this qa warning:"


ImplementationGuide/hl7.fhir.uv.smart: ImplementationGuide | warning | ig-0: 'Name should be usable as an identifier for the module by machine processing applications such as code generation' failed

because of:


inv-0 | Warning | Name should be usable as an identifier for the module by machine processing applications such as code generation | name.matches('[A-Z]([A-Za-z0-9_]){0,254}')


so I just Make them PascalCase